onnxruntime/core/providers/cuda/generator/random.h

```diff
@@ -26,6 +26,7 @@ RANDOM_COMPUTE_IMPL(RandomUniform)
 
```

Comment:
These are used only in the .cc file. Let's move it there. Implementation detail.